conference/management/commands/create_initial_data_for_dev.py

```diff
@@ -24,103 +24,163 @@ class Command(BaseCommand):
+        #     placeholder=homepage.placeholders.get(slot="lead_text"),
+        #     plugin_type=TextPlugin,
+        #     language="en",
+        #     body="This is the lead text",
+        # )
+        # add_plugin(
+        #     placeholder=homepage.placeholders.get(slot="home_teaser_text"),
+        #     plugin_type=TextPlugin,
+        #     language="en",
+        #     body="This is the home teaser text",
+        # )
         # The main_text placeholder does not seem to be used, skipping.
 
         print(
             "Created page: ",
             homepage.reverse_id,
             homepage.title_set.first().title,
-            homepage.template
+            homepage.template,
         )
 
-        pages = [
-            ('contacts', 'CONTACTS', 'content.html'),
-            ('privacy', 'PRIVACY', 'content-1col.html'),
-            ('conduct-code', 'CONDUCT-CODE', 'content.html'),
-            ('staff', 'STAFF', 'content.html'),
-            ('sponsor', 'SPONSOR', 'content.html'),
-        ]
-
-        for id, title, template in pages:
-
+        def new_page(rev_id, title, **kwargs):
             try:
                 page = create_page(
+                    reverse_id=rev_id,
                     title=title,
-                    template='django_cms/' + template,
                     language='en',
-                    reverse_id=id,
+                    template=(
+                        'ep19/bs/content/'
+                        'generic_content_page_with_sidebar.html'
+                    ),
                     published=True,
                     publication_date=timezone.now(),
                     in_navigation=True,
+                    **kwargs
                 )
                 print("Created page: ", page.reverse_id, title, page.template)
+                return page
 
             # FieldError happens to be what django cms is using when we want to
             # create another page with the same reverse_id
             except FieldError as e:
                 print("Warning: ", e)
 
+        program_page = new_page('program', "Program")
+
+        for rev_id, title in [
+            ("speakers", "Speakers"),
+            ("tranings", "Tranings"),
+            ("workshops", "Workshops"),
+            ("talks_and_conference_days", "Talks and Conference Days"),
+            ("social-event", "Social Event"),
+            ("sprints", "Sprint"),
+        ]:
+            new_page(rev_id, title, parent=program_page)
+
+        location_page = new_page('location', "Location")
+        for rev_id, title in [
+            ("explore-basel", "Explore City of Basel"),
+            ('visa', "Visa"),
+            ('conference-venue', "Conference Venue"),
+            ('sprints-venue', "Workshops & Sprints Venue"),
+        ]:
+            new_page(rev_id, title, parent=location_page)
+
+        about_europython_page = new_page("about", "About EuroPython")
+        for rev_id, title in [
+            ('volnuteers', 'Volunteers'),
```

Comment:
There's a typo in the first tuple element - `volunteers`

Reply:
:+1: